decidim-elections/app/forms/decidim/elections/admin/trustees_participatory_space_form.rb

```diff
@@ -16,6 +17,12 @@ def map_model(trustee)
         def user
           @user ||= current_organization.users.find_by(id: user_id)
         end
+
+        def unique_trustee_name
+          if Decidim::Elections::Trustee.joins(:user).where(decidim_users: { decidim_organization_id: current_organization.id }).where(name: user.name).any?
```

Comment:
💬 maybe using `exists?` the query will be faster, as you don't need to retrieve the trustee, just check if it exists or not in the database. You can do something like:
```ruby
if Decidim::Elections::Trustee.joins(:user).exists?(name: user.name, decidim_users: { decidim_organization_id: current_organization.id })
```